src/django_upgrade/fixers/settings_database_posgresql.py

```diff
@@ -0,0 +1,50 @@
+"""
+Update settings.DATABASES backend path 'django.db.backends.postgresql_psycopg2'
+to the new 'django.db.backends.postgresql'.
+https://docs.djangoproject.com/en/2.0/releases/2.0/#id1
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable
+
+from tokenize_rt import Offset
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer, State, TokenFunc
+from django_upgrade.tokens import replace
+
+fixer = Fixer(
+    __name__,
+    min_version=(1, 9),
+)
+
+
+@fixer.register(ast.Dict)
+def visit_Dict(
+    state: State,
+    node: ast.Dict,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        isinstance(parents[-1], ast.Dict)
+        and isinstance((db_setting := parents[-2]), ast.Assign)
+        and isinstance(db_setting.targets[0], ast.Name)
+        and db_setting.targets[0].id == "DATABASES"
+        and len(db_setting.targets) == 1
+        and any(
+            (
+                isinstance(key, ast.Constant)
+                and key.value == "ENGINE"
+                and isinstance(val, ast.Constant)
+                and (target_node := val).value
+                == "django.db.backends.postgresql_psycopg2"
+            )
+            for key, val in zip(node.keys, node.values)
+        )
+        and state.looks_like_settings_file()
+    ):
+        yield ast_start_offset(target_node), partial(
+            replace, src='"django.db.backends.postgresql"'
+        )
```

Comment:
since #260, django-upgrade tries to match the quoting choice of existing code, so updating this to use a small token function to match the existing quote char

Reply:
ah yeah right, I've seen in the blog post you merge that in the end. mb